libraries/engage/page/widgets/CategoryList/hooks.js

```diff
@@ -31,6 +33,8 @@ export const useCategoryListWidget = () => {
     category,
     sort,
     showImages,
+    showHeadline = false,
+    headline = '',
```

Comment:
please remove the default value since it's not necessary. additionally it's kinda wrong, since `headline` is usually an object. so the default shouldn't be a string.